### PR TITLE
Pill Packet QOL

### PR DESCRIPTION
**Pull request by PhantomEpicness**
## About The Pull Request
A small QoL PR for pill packets. 
The amount of pills left in them is now reflected in sprites, furthermore, empty pill packets are now trash.

Screenshot_424
Screenshot_425

## Why It's Good For The Game
soul

## Changelog

:cl: TotalEpicness
qol: Pill packets now reflect the remaining amount of pills.
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -643,6 +643,19 @@ obj/item/storage/pill_bottle/packet
 	skilllock = SKILL_MEDICAL_DEFAULT
 	storage_flags = STORAGE_FLAGS_BOX
 	display_maptext = FALSE
+	var/icon_original
+
+/obj/item/storage/pill_bottle/packet/Initialize()
+	. = ..()
+	icon_original = icon_state
```
Comment (morrowwolf) [inline code/game/objects/items/storage/firstaid.dm:650]: This looks like it could be done with initial() rather than a var. Let me know if I am incorrect.

**Review by morrowwolf — CHANGES_REQUESTED** [review]

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Getting some runtimes related to the qdel.

**Comment by PhantomEpicness** [comment]
runtimes fixed

**Comment by PhantomEpicness** [comment]
nvm forgot to push from desktop at home

**Review by morrowwolf — CHANGES_REQUESTED** [review]
This doesn't need to touch parent code to do this. This has a lot of implications that are unnecessary.

Override the attempt remove function from /obj/items/storage in /obj/item/storage/pill_bottle/packet and try getting that to work.

Should be a simple:
```
/obj/item/storage/pill_bottle/packet/whatever_the_remove_proc_is(arguments)
..() or if(..()) as applicable
check for nothing in contents then make your trash and qdel(src)
```

**Comment by PhantomEpicness** [comment]
> 



> This doesn't need to touch parent code to do this. This has a lot of implications that are unnecessary.
> 
> Override the attempt remove function from /obj/items/storage in /obj/item/storage/pill_bottle/packet and try getting that to work.
> 
> Should be a simple:
> 
> ```
> /obj/item/storage/pill_bottle/packet/whatever_the_remove_proc_is(arguments)
> ..() or if(..()) as applicable
> check for nothing in contents then make your trash and qdel(src)

yeah when I was tweakin while making this

fixed

**Review by Agameofscones — CHANGES_REQUESTED** [review]
You should handle the individual sprites as a greyscale overlay. This amount of hardsprites is unacceptable and makes pill packets less modular for future content development

**Pull request closed without merging**